src/api/spec/features/webui/requests_spec.rb

```diff
@@ -1,10 +1,27 @@
 require "browser_helper"
+# WARNING: If you change tests make sure you uncomment this line
+# and start a test backend. Some of the BsRequestAction methods
+# require real backend answers for projects/packages.
+# CONFIG['global_write_through'] = true
 
 RSpec.feature "Requests", :type => :feature, :js => true do
-  RSpec.shared_examples "expandable element" do
-    let!(:bs_request) { create(:bs_request, description: "a long text - " * 200) }
-    let!(:user) { create(:confirmed_user) }
+  let!(:submitter) { create(:confirmed_user, login: 'kugelblitz' ) }
+  let!(:receiver) { create(:confirmed_user, login: 'titan' ) }
```

Comment:
In this case and many others variables defined using `let!` is better to define them just using `let` because those variables are unused in many spec examples. `let!` is creating/running the variable for every example in the feature spec.